wrangler-core/src/test/java/io/cdap/directives/parser/ParseAvroFileTest.java

```diff
@@ -52,6 +52,40 @@ public void testParseAsAvroFile() throws Exception {
     Assert.assertEquals(1495194308245L, results.get(1688).getValue("timestamp"));
   }
 
+  @Test
+  public void testAvroWithJsonFieldParsingWithParseJsonIgnoreError() throws Exception {
+    InputStream stream = ParseAvroFileTest.class.getClassLoader().getResourceAsStream("bad.avro");
+    byte[] data = IOUtils.toByteArray(stream);
+
+    String[] directives = new String[] {
+      "parse-as-avro-file body",
+      "parse-as-json :message_contents 2 true"
+    };
+
+    List<Row> rows = new ArrayList<>();
+    rows.add(new Row("body", data));
+
+    List<Row> results = TestingRig.execute(directives, rows);
+    Assert.assertEquals(6693, results.size()); // total 6670, 7 records are bad.
```

Comment:
these types of asserts are not specific enough. For example, how can we be sure the right rows are filtered? It would be better to just send a single row in as input.

Reply:
Using customer test data to make sure we don't break it.

Reply:
this doesn't make sure of anything besides the size